docker/docker-chaos/metrictank.ini

```diff
@@ -362,17 +362,17 @@ hosts = cassandra:9042
 protocol-version = 4
 # write consistency (any|one|two|three|quorum|all|local_quorum|each_quorum|local_one
 consistency = one
-# cassandra request timeout
+# cassandra request timeout. valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'
 timeout = 1s
 # number of concurrent connections to cassandra
 num-conns = 10
 # Max number of metricDefs allowed to be unwritten to cassandra
 write-queue-size = 100000
-#Interval at which the index should be checked for stale series.
+#Interval at which the index should be checked for stale series. valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'
 prune-interval = 3h
 # synchronize index changes to cassandra. not all your nodes need to do this.
 update-cassandra-index = true
-#frequency at which we should update flush changes to cassandra. only relevant if update-cassandra-index is true.
+#frequency at which we should update flush changes to cassandra. only relevant if update-cassandra-index is true. valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'. Setting to '0s' will cause instant updates.
```

Comment:
kindof weird to tell users that they can use all these subsequent units but in `New()` round down to a round number of seconds.  but it's fine i guess.
we could use https://github.com/raintank/dur instead which accepts units `/sec/secs/second/seconds, m/min/mins/minute/minutes, h/hour/hours, d/day/days, w/week/weeks, mon/month/months, y/year/years` but those high units are more than what we need. a side benefit would be though that we don't need to convert to uint32 anymore.
anyway, seems like more hassle than it's worth.

Reply:
You are right, that is a little weird. Maybe we should specify that although you can use all of those time units, it will round to 1 second. Also, if the time they specify is less than 1 second that it will treat it like 0, thus causing instant updates.

Reply:
I think we don't have to do anything. our performance with value of 1s would be the same as it is with <1s since we don't accept messages more frequently than a second anyway.